Add new external data template (#238)
GeoscienceAustralia/dea-knowledge-hub: docs/_templates/external-data-v1.rst

            - Resolution
            - CRS
            - Nodata
            - Type
            - Description
          {% for band in valid_bands %}
          * - **{{ band.name }}**
            - {{ band.aliases|join(', ') if band.aliases else none_text }}
            - {{ band.resolution or not_available_text }}
            - {{ band.crs or not_available_text }}
            - {{ band.nodata }}
            - {{ band.type or not_available_text }}
            - {{ band.description or none_text }}
          {% endfor %}

       {{ Specifications.bands.footnotes if Specifications.bands.footnotes }}
       {% endif %}
    {% endif %}

    {% if Data.enable_access %}
    .. tab-item:: Access
       :name: access

       .. raw:: html

          <div class="product-tab-table-of-contents"></div>

       .. rubric:: Access the data
          :name: access-the-data-2
          :class: h2

       {% if has_access_data %}
       .. list-table::
          :name: access-table

          {% if valid_external_data %}
          * - **{{ external_data_label }}**
            - 
              * `{{ valid_external_data.custom_label or external_data_default_name }} <{{ valid_external_data.link }}>`_

>             - The external page should contain information about how to access the data.

Reviewer: Access the the external pages is probably more about users being able to learn more about the data vs. purely data access (given we're providing our own data access options via DEA's systems). Some possible alternative wordings:

```suggestion
            - Access additional information about this dataset from the external provider.
```

```suggestion
            - Learn more about this data from the external provider.
```

Reviewer 2: > Learn how to access the data from the external provider.

Reviewer: The "how to access the data" is the bit we don't want to include - we want users to access data using DEA systems, and the external provider's page won't provide details on how to load the data in a way that's relevant to DEA.

The purpose of these links is to allow users to find out more information about the products from the provider, allowing us to make our listings minimal and not reproduce that material.

Reviewer 2: Ok very good point -- I've now reworded it to

> Learn more about the data from the external provider.

   .. admonition:: External data
      :class: note external-data
   

>       This data product is developed by an external party ({{ Data.external_provider }}), and is not a DEA product. The data hasn't been altered except to make it compatible with DEA's systems.

Reviewer: Slight wording update to be less colloquial:

> The data **has not** been altered except to make it compatible with DEA's systems.

Reviewer 2: Latest:

> This data product is produced by an external party ({{ Data.external_provider }}), and is not a DEA product. DEA provides this data without modifications except where needed to make it compatible with our systems.
